README.rst

```diff
@@ -1,121 +0,0 @@
-.. image:: https://travis-ci.org/iotaledger/iota.py.svg?branch=master
```

Comment:
README.rst is needed to build the documentation for ReadTheDocs.
We could move this file under `docs` folder, and modify `docs/index.rst` to contain
```
.. include:: README.rst
```